package/network/ipv6/map/files/map.sh

```diff
@@ -223,6 +223,7 @@ proto_map_init_config() {
 	proto_config_add_int "ealen"
 	proto_config_add_int "psidlen"
 	proto_config_add_int "psid"
+	proto_config_add_int "ealen"
```

Comment:
This added line is similar to the line 3 above (https://github.com/openwrt/openwrt/pull/771/files#diff-ef12204fd388a1a753727c00529f6afeR223); so the added line seems unnecessary unless I miss something ?

Reply:
Doh, sorry, someone else must have already fixed it before I send my patch. Please ignore!